pdks/Gf180/gf180_hdl21/pdk_data.py

```diff
@@ -62,34 +62,50 @@ class MosParams:
     w = h.Param(dtype=h.Scalar, desc="Width in PDK Units (µm)", default=1 * µ)
```

Comment:
This is copied from Sky130, AFAIK, GF180 does actually use SI units - so mathematically this checks out, but yes, the description is silly.

Reply:
Ah, okay great! Perhaps we could change it to something like this then?

```python
w = h.Param(dtype=h.Scalar, desc="Width in PDK Units (m)", default=720 * n)
l = h.Param(dtype=h.Scalar, desc="Length in PDK Units (m)", default=180 * n)
```

Do you know what the minimum size devices are in the standard xtor Pcells for the GF180 PDK? I have `720 nm` and `180 nm` above, but I'm just guessing.

Reply:
Hey, [this documentation](https://gf180mcu-pdk.readthedocs.io/en/latest/analog/model_parameters/LV/LV_2_2.html) shows sizing for standard pcells - otherwise, the documentation doesn't do a fantastic job of keeping track of the minimum sizing and I just borrow values used in the xschem demonstrator, which is as good as guessing :shrug: